napalm_yang/mappings/eos/parsers/state/openconfig-interfaces/interfaces.yaml

```diff
@@ -27,82 +28,68 @@ interfaces:
             admin-status:
                 _process:
                     - mode: map
-                      value: "{{ bookmarks.parent.get('interfaceStatus', 'disabled') }}"
+                      path: interfaceStatus
+                      defaul: disabled
```

Comment:
Yes, thanks! I am actually going to remove the `default` argument. I don't think it's needed.